Makefile

```diff
@@ -235,8 +235,15 @@ $(EXES):
 protos: ## Generates protobuf files.
 protos: $(PROTO_GOS)
 
+GENERATE_FILES ?= true
```

Comment:
Should we call it GENERATE_PROTOBUF_FILES instead?

Reply:
I was thinking we could use the same variable for other kinds of files (if we ever have any) in the future - but happy to change it if you feel strongly about it.

Reply:
I don't feel strong.